### PR TITLE
fix(testimonials): Fixed alt text accessibility 

### LINKED ISSUE #51033: Make alt text on alumni images more accessible
### Describe the Issue

This is a minor issue, but I think it needs to be addressed. The current alt text for the three alumni images on the  main landing page is just the person's name. This is merely duplicating information that is already provided and doesn't give people who can't see the image the same information as people who can see the image.

### Affected Page

https://www.freecodecamp.org/

### Steps to Reproduce

1. Go to 'https://www.freecodecamp.org/'
2. View alt text for each alumni image with the browser's dev tools inspector

### Expected behavior

Alt text for each image should be updated to be a description of the image. This doesn't need to be extremely detailed. Just hit the basic details. Perhaps we could reach out to each of these people and ask them to write their own alt text?

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DylanDevelops**
- Updated alt text for testimony images
- Updated caption for the study group

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #51033

I noticed that the old pull request was closed, so I went ahead and fixed everything in this one pull request.

Also, please let me know if we don't want to update the caption of the study group. It was discussed in the issue but never fully decided upon like the others.

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
> Also, please let me know if we don't want to update the caption of the study group. It was discussed in the issue but never fully decided upon like the others.

It was discussed in [this comment](https://github.com/freeCodeCamp/freeCodeCamp/pull/51042#issuecomment-1678475938).

The `hero-img-description` should not be changed.
A new `hero-img-alt` property should be added with the text `A group of people, including a white man, a black woman, and an Asian woman, gathered around a laptop.`

Then, the [alt attribute](https://github.com/freeCodeCamp/freeCodeCamp/blob/05eab884b0dccc896e9eff3ba50b923b0e31cf51/client/src/components/landing/components/campers-image.tsx#L32) should be updated to use the new `hero-img-alt` translation value.

**Comment by DylanDevelops** [comment]
Do you think this looks right? Sorry, I missed the talk about the alt text in the main image. 😅

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -92,6 +92,7 @@
     "big-heading-3": "Earn certifications.",
     "h2-heading": "Since 2014, more than 40,000 freeCodeCamp.org graduates have gotten jobs at tech companies including:",
     "hero-img-description": "freeCodeCamp students at a local study group in South Korea.",
+    "hero-img-alt": "A group of people, including a white man, a black woman, and an Asian woman, gathered around a laptop.",
```
Comment (naomi-lgbt) [inline client/i18n/locales/english/translations.json:95]:
```suggestion
    "hero-img-alt": "A group of people, including a White man, a Black woman, and an Asian woman, gathered around a laptop.",
```

Nit: We should probably capitalise these like we did in the other alt texts.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]
Happy to approve this. Thanks for your work!

**Review by raisedadead — APPROVED** [review]

**Review by bbsmooth — APPROVED** [review]
I'm good with these if you all are.

**Comment by naomi-lgbt** [comment]
Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**